hnn_core/tests/test_optimize_evoked.py

```diff
@@ -170,9 +172,9 @@ def test_optimize_evoked():
     assert drive_static_params_opt == drive_static_params_orig
 
     # ensure that only the drive that we wanted to optimzie over changed
-    drive_evdist1_dynamics_offset, drive_evdist1_syn_weights_offset,\
+    drive_evdist1_dynamics_offset, drive_evdist1_syn_weights_offset, \
```

Comment:
Quick heads up, @raj1701 had to change this in #661 as well because of the flake8 update (so you might get a merge conflict during the rebase)

Reply:
Guess it depends on who's PR is merged first, sorry we've unintentionally turned this into a race :smile: